ref: document `dvc queue` (#3715)
iterative/dvc.org: content/docs/command-reference/exp/run.md

  parallel. Only has an effect along with `--run-all`. Defaults to 1 (the queue
  is processed serially).

>   > ⚠️ `dvc exp run --run-all [--jobs]` is now a shortcut for
>   > `dvc queue start [--jobs]` followed by `dvc queue logs -f`. The `--run-all`
>   > and `--jobs` options will likely be deprecated and removed in a future DVC
>   > release. It is recommended to migrate your workflows to use
>   > `dvc queue start` and `dvc queue logs`. Refer to the `dvc queue`
>   > documentation for more details.

Reviewer: Making proper admon

```suggestion
  <admon type="warn">
  
  `dvc exp run --run-all [--jobs]` is now a shortcut for
  `dvc queue start [--jobs]` followed by `dvc queue logs -f`. The `--run-all`
  and `--jobs` options will be deprecated in a future DVC
  release.

  </admon>
```

  parallel. Only has an effect along with `--run-all`. Defaults to 1 (the queue
  is processed serially).

  <admon type="warn">

  `dvc exp run --run-all [--jobs]` is now a shortcut for
  `dvc queue start [--jobs]` followed by `dvc queue logs -f`. The `--run-all`
  and `--jobs` options will be deprecated in a future DVC release.

>   </admon>

Reviewer: Maybe this should be under `--run-all` ? Or split into 2 admonitions, even if they're similar.

Reviewer 2: @pmrowla What do you think about moving or splitting this one?

Author: I don't really think it's necessary to repeat this block after 2 consecutive options (`run-all` and `jobs` will appear one after another and both options + the info block fit into a single screen on most devices

Reviewer: True. But this is much more about `--run-all` than about `--jobs` so I moved it up in https://github.com/iterative/dvc.org/pull/3894/commits/26d28e8b4dae92ff3100b7a8ebf5d2bfbd36272a.

>   > stages may sometimes be executed several times depending on the state of the
>   > [run-cache] at that time.

Reviewer: Removing this note since it's in the `queue run` ref now... (1/2)

```suggestion
```

  > ⚠️ `dvc exp run --run-all [--jobs]` is now a shortcut for
  > `dvc queue start [--jobs]` followed by `dvc queue logs -f`. The `--run-all`
  > and `--jobs` options will likely be deprecated and removed in a future DVC
  > release. It is recommended to migrate your workflows to use
  > `dvc queue start` and `dvc queue logs`. Refer to the `dvc queue`
  > documentation for more details.

> 

Reviewer 3: Should we also include this kind of warning in core DVC (i.e. actually logging `DeprecationWarning`)?

Reviewer 2: It's fine to say that this will be deprecated (we are already doing it here). There shouldn't be a hard requirement to include deprecation of experiment features since they are "experimental" in 2.X, but it would be nice.
